src/views/Playgrounds/PlaygroundChannelEditCreate.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useRef, useState, useEffect } from 'react'
+import { useForm, Controller } from 'react-hook-form'
+
+import { ImageCropDialog, ImageCropDialogImperativeHandle, ViewWrapper } from '@/components'
+import { ChannelCover, FormField, Select, SelectedItem, HeaderTextField, Tooltip } from '@/shared/components'
+
+import { transitions } from '@/shared/theme'
+import {
+  StyledTitleSection,
+  TitleContainer,
+  StyledFormField,
+  StyledTextarea,
+  StyledActionBarTransaction,
+  StyledAvatar,
+  InnerFormContainer,
+} from './PlaygroundChannelEditCreate.style'
+import { Header, SubTitle, SubTitlePlaceholder, TitlePlaceholder } from '../ChannelView/ChannelView.style'
+
+import { requiredValidation, textFieldValidation } from './formValidationOptions'
+import { formatNumberShort } from '@/utils/number'
+
+const languages: SelectedItem[] = [
```

Comment:
We will need to populate this with some more languages. Also, the key should probably be the ISO 639-1 identifier of the language as that's what's in the schema. I suggest to find some list of the most popular languages and put the first 20 or so here. We can revisit later